<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dtgConfig</class>
 <widget class="QWidget" name="dtgConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>455</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0">
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Font</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Font style:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>plainDateTimeGroupFont</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="KFontComboBox" name="plainDateTimeGroupFont"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="2">
    <widget class="QCheckBox" name="plainDateTimeGroupFontBold">
     <property name="toolTip">
      <string>Check if you want the font in bold</string>
     </property>
     <property name="whatsThis">
      <string>When this is checked, the date time group font will be bold.</string>
     </property>
     <property name="text">
      <string>&amp;Bold</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QCheckBox" name="plainDateTimeGroupFontItalic">
     <property name="toolTip">
      <string>Check if you want the font in italic</string>
     </property>
     <property name="whatsThis">
      <string>When this is checked, the date time group font will be in italic.</string>
     </property>
     <property name="text">
      <string>&amp;Italic</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="horizontalSpacer_9">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Custom font color:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>useCustomColor</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QCheckBox" name="useCustomColor">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KColorButton" name="plainDateTimeGroupColor">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Color chooser</string>
       </property>
       <property name="whatsThis">
        <string>Click on this button and the KDE standard color dialog will show. You can then choose the new color you want for your date time group.</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Show shadow:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>drawShadow</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="drawShadow">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_13">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="customShadowColorLabel">
     <property name="text">
      <string>Custom shadow color:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>useCustomShadowColor</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="1">
      <widget class="KColorButton" name="plainDateTimeGroupShadowColor">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Shadow color chooser</string>
       </property>
       <property name="whatsThis">
        <string>Click on this button and the KDE standard color dialog will show. You can then choose the new color you want for the text shadow for your date time group.</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer_12">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="useCustomShadowColor">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QLabel" name="label_7">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Information</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLabel" name="label_4">
     <property name="toolTip">
      <string>Use Z time (UTC). Changes the current time zone.</string>
     </property>
     <property name="text">
      <string>NATO operation:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>natoOperation</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="natoOperation">
       <property name="toolTip">
        <string>Use Z time</string>
       </property>
       <property name="whatsThis">
        <string>Use Z time - as in NATO operations.</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="10" column="1">
    <widget class="QLabel" name="label_5">
     <property name="toolTip">
      <string>Use local time and the NATO identifier J. Changes the current time zone.</string>
     </property>
     <property name="text">
      <string>Local time:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="10" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QCheckBox" name="localTime">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="11" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Format</string>
     </property>
    </widget>
   </item>
   <item row="18" column="1">
    <widget class="QLabel" name="label_10">
     <property name="toolTip">
      <string>Just show the time (HHMMZ).</string>
     </property>
     <property name="text">
      <string>Short format:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="18" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QRadioButton" name="shortFormat">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="19" column="1">
    <widget class="QLabel" name="label_11">
     <property name="toolTip">
      <string>Show the standard date time group (DDHHMMZMMMYY)</string>
     </property>
     <property name="text">
      <string>Standard format:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="19" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QRadioButton" name="stanFormat">
       <property name="text">
        <string/>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="20" column="1">
    <widget class="QLabel" name="label_13">
     <property name="toolTip">
      <string>Show the longest date time group format (DDHHMMSSZMMMYYYY)</string>
     </property>
     <property name="text">
      <string>Long format:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="20" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QRadioButton" name="longFormat">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="25" column="2">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>2</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="22" column="0">
    <widget class="QLabel" name="label_8">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Spaces</string>
     </property>
    </widget>
   </item>
   <item row="23" column="2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>after day</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QCheckBox" name="sAfterDay">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QCheckBox" name="sAfterHour">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>after hours</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>after minutes</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>after seconds</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QCheckBox" name="sAfterMinute">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QCheckBox" name="sAfterSecond">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>after time zone</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>after month</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QCheckBox" name="sAfterTimeZone">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QCheckBox" name="sAfterMonth">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="23" column="1">
    <widget class="QLabel" name="label_17">
     <property name="text">
      <string>Add spaces:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KFontComboBox</class>
   <extends>KComboBox</extends>
   <header>kfontcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>useCustomColor</sender>
   <signal>toggled(bool)</signal>
   <receiver>plainDateTimeGroupColor</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useCustomShadowColor</sender>
   <signal>toggled(bool)</signal>
   <receiver>plainDateTimeGroupShadowColor</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
